website/src/data/users.tsx

```diff
@@ -314,7 +324,7 @@ const Users: User[] = [
       website: 'https://github.com/Azure-Samples',
       author: 'Azure Content Team',
     source: 'https://github.com/Azure-Samples/svc-invoke-dapr-csharp',
-    tags: ['dapr','csharp','azurecontainerapps']
```

Comment:
Why change to aca? Not sure that will mean anything to people filtering...

Reply:
This doesn't change the publicly visible tag. The tag in the site should be "Azure Container Apps", this is just what's currently being used on the backend.